### PR TITLE
Division / Sector / Branch label order change

### PR DESCRIPTION
**Pull request by wangf1122**
Change the label order to Division / Sector / Branch

### CODE REVIEW

**Comment by josegar74** [comment]
From HNAP 2.3.1 specification: 

> Guideline:  organizationName shall be mandatory. Federal department and agency titles will conform to Federal Identity Program policy and use the Applied Title for their department or agency as defined at http://www.tbs-sct.gc.ca/fip-pcim/reg-eng.asp. 5.16.2 organizationName will conform to the format:  Government of Canada; AppliedTitle. Sub-organizations (sectors, branches, etc.) may be included at the option of the department. See Responsible Party example below.

**Comment by ianwallen** [comment]
Suggested changes 

Branch, Section, Division, etc...

**Comment by ianwallen** [comment]
Related to #289

**Comment by wangf1122** [comment]
> Suggested changes
> 
> Branch, Section, Division, etc...

I have made the changes. Please bring it up to the next community meeting see if this label fits.

File: src/main/plugin/iso19139.ca.HNAP/layout/layout-custom-fields.xsl [hunk 1]
```diff
@@ -235,8 +235,8 @@
           {
             "type": "freeText",
             "heading": {
-              "eng": "Branch/Sector/Division",
-              "fra": "Branche/Secteur/Division"
+              "eng": "Branch, Section, Division, etc...",
+              "fra": "Branche, Secteur, Division, etc..."
```
Comment (ianwallen) [inline src/main/plugin/iso19139.ca.HNAP/layout/layout-custom-fields.xsl]: Translation is the following

`Branche, Section, Division, etc...`

**Comment by ianwallen** [comment]
Looking at the text directly from the hnap specs it may be better to use their wording?

```
Sub-organizations (sectors, branches, etc.)
Sous-organisations (secteurs, branches, etc.)
```

**Comment by wangf1122** [comment]
> Sub-organizations (sectors, branches, etc.)

no problem, its been updated.

**Review by ianwallen — APPROVED** [review]

**Pull request merged**